validator/inputs/writer.go

```diff
@@ -132,9 +146,13 @@ func (w *Writer) Write(json *server_api.InputJSON) error {
 	if err != nil {
 		return err
 	}
-	if err = os.WriteFile(
-		filepath.Join(dir, fmt.Sprintf("block_inputs_%d.json", json.Id)),
-		contents, 0600); err != nil {
+	var fileName string
```

Comment:
Nit: I'd probably write this like:

fileName := "block_inputs.json"
if w.useBlockIdInFileName {
    fileName = fmt.Sprintf("block_inputs_%d.json", json.Id)
}
if err = os.WriteFile(filepath.Join(dir, fileName), contents, 0600); err != nil {
    return err
}

The reason is that it makes it a little more clear exactly what the useBlockIdInFileName switch is changing. I can see right away that it's just inserting the Id from the json file into the fileName. Before, as someone reading the code for the first time, I have to compare the two lines and think, "Is it changing how the filepath.Join() function is called? Oh, no, it's not. Those are the same, so it must be just a change in the last element of the path. Ah, yes, it's putting the Id in there."